core/src/test/java/org/elasticsearch/monitor/jvm/JvmInfoTests.java

```diff
@@ -58,4 +62,38 @@ private boolean flagIsEnabled(String argline, String flag) {
         return argline.charAt(index - 1) == '+';
     }
 
+    public void testSerialization() throws IOException {
+        JvmInfo jvmInfo = JvmInfo.jvmInfo();
```

Comment:
I think it'd be nice to be able to make a random one? Maybe in a package private ctor?